engine/action_contact.go

```diff
@@ -163,6 +163,62 @@ func (c *Client) UpdateContact(ctx context.Context, id, fullName string, metadat
 	return contact, nil
 }
 
+// AdminCreateContact creates a new contact - xpubId is retrieved by the creatorPaymail.
+func (c *Client) AdminCreateContact(ctx context.Context, contactPaymail, creatorPaymail, fullName string, metadata *Metadata) (*Contact, error) {
```

Comment:
should you validate inputs like `contactPaymail, creatorPaymail, fullName` ? or they can be empty

Reply:
Explained in previous comment. Now they are validated (contactPaymail is validated at the beginning:
```go
	contactPaymail := c.Param("paymail")
	if contactPaymail == "" {
		spverrors.ErrorResponse(c, spverrors.ErrMissingContactPaymailParam, logger)
		return
	}
```